Description: Do not install potentials
   The files are installed directly from the source
Author: Anton Gladky <gladk@debian.org>
Last-Update: 2021-11-11

Index: lammps/cmake/CMakeLists.txt
===================================================================
--- lammps.orig/cmake/CMakeLists.txt	2024-02-18 13:21:15.604614137 +0100
+++ lammps/cmake/CMakeLists.txt	2024-02-18 13:21:15.592614031 +0100
@@ -800,8 +800,8 @@
 ###############################################################################
 set(LAMMPS_INSTALL_DATADIR ${CMAKE_INSTALL_DATADIR}/lammps)
 
-install(DIRECTORY ${LAMMPS_DIR}/bench DESTINATION ${LAMMPS_INSTALL_DATADIR})
-install(DIRECTORY ${LAMMPS_POTENTIALS_DIR} DESTINATION ${LAMMPS_INSTALL_DATADIR})
+#install(DIRECTORY ${LAMMPS_DIR}/bench DESTINATION ${LAMMPS_INSTALL_DATADIR})
+#install(DIRECTORY ${LAMMPS_POTENTIALS_DIR} DESTINATION ${LAMMPS_INSTALL_DATADIR})
 if(BUILD_TOOLS)
   install(DIRECTORY ${LAMMPS_TOOLS_DIR}/msi2lmp/frc_files DESTINATION ${LAMMPS_INSTALL_DATADIR})
 endif()
